### PR TITLE
🏇 Update bootstrap.sh

### LINKED ISSUE #41: 📝 Upload The GUIDE For "Live-Refresh" As Markdown
Maybe we should even add this to our [docs](https://github.com/redhat-beyond/JobSeeker/tree/main/docs) folder, to help other contributors.

_Originally posted by @taljacob2 in https://github.com/redhat-beyond/JobSeeker/issues/38#issuecomment-1073054820_

---

~~Upload the GUIDE for "Live-Refresh" that is in #38, as a markdown in the [docs](https://github.com/redhat-beyond/JobSeeker/tree/main/docs) folder, to help other contributors.~~

**UPDATE:** Instead of creating a markdown for the "workaround" solution in #38, we will use [Barak's fix for this issue](https://github.com/redhat-beyond/JobSeeker/issues/41#issuecomment-1073349042).

### LINKED ISSUE #54: 🏇 Automating migrations in Vagrant

### PR DESCRIPTION
**Pull request by DeanBiton**
- Added the following command:
    "pipenv run python manage.py migrate"
- Changed the manage.py runserver command

close #41 
close #54

### CODE REVIEW

File: bootstrap.sh [hunk 1]
```diff
@@ -10,6 +10,9 @@ cd /vagrant
 # Install dependencies with Pipenv
 pipenv sync --dev
 
+# Run database migrations
+pipenv run python manage.py migrate
+
 # run our app. Nohup and “&” are used to let the setup script finish
 # while our app stays up. The app logs will be collected in nohup.out
 nohup pipenv run python manage.py runserver 0.0.0.0:8000 &
```
Comment (Yarboa) [inline bootstrap.sh:18]: Please update this line as per the discussion in os-07 slack channel

Comment (taljacob2) [reply bootstrap.sh:18]: @DeanBiton Hi, edit this file according to [Barak's commit](https://github.com/redhat-beyond/JobSeeker/issues/41#issuecomment-1073349042).
Once you do it, please state in this PR's description:
```
Close #41
```

**Review by taljacob2 — APPROVED** [review]
LGTM!

**Review by paOmer — APPROVED** [review]
LGTM!

**Review by amittcohen — APPROVED** [review]
LGTM

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**